nx-libs.spec

```diff
@@ -1,4 +1,8 @@
 %global _hardened_build 1
+# prevent "ERROR: No build ID note found" errors
+%if 0%{rhel} < 7
```

Comment:
Needs fixup: `%if 0%{?rhel} && 0%{?rhel} < 7`

(For once, missing the `?` char which will cause errors when the macro is not defined, secondly `0%{?rhel} < 7` will evaluate to true if `%{rhel}` is undefined [since `0%{?rhel} < 7` becomes `0 < 7` in that case and this is `true`], which is not what we want.)